packages/devtools_app/lib/src/screens/network/har_network_data.dart

```diff
@@ -67,31 +75,28 @@ class HarDataEntry {
   factory HarDataEntry.fromJson(Map<String, Object?> json) {
     _convertHeaders(json);
 
-    final modifiedRequestData =
-        _remapCustomFieldKeys(json) as Map<String, dynamic>;
+    final modifiedRequestData = _remapCustomFieldKeys(json);
 
     // Retrieving url, method from requestData
-    modifiedRequestData['uri'] =
-        (modifiedRequestData['request'] as Map<String, Object?>)['url'];
-    modifiedRequestData['method'] =
-        (modifiedRequestData['request'] as Map<String, Object?>)['method'];
+    final requestData = modifiedRequestData[NetworkEventKeys.request.name]
+        as Map<String, Object?>;
+    modifiedRequestData[NetworkEventKeys.uri.name] =
+        requestData[NetworkEventKeys.url.name];
+    modifiedRequestData[NetworkEventKeys.method.name] =
+        requestData[NetworkEventKeys.method.name];
 
     // Adding missing keys which are mandatory for parsing
-    (modifiedRequestData['response'] as Map<String, Object?>)['redirects'] = [];
+    final responseData = modifiedRequestData[NetworkEventKeys.response.name]
+        as Map<String, Object?>;
+    responseData[NetworkEventKeys.redirects.name] = <Map<String, Object?>>[];
     Object? requestPostData;
```

Comment:
You can set these values directly instead of doing a null check later:
`final requestPostData = responseData[NetworkEventKeys.postData.name];`.

This is because `requestPostData` will still be set to null if `responseData[NetworkEventKeys.postData.name]` is null. 

Same for the `responseContent` variable.

Reply:
makes sense, updated